selfdrive/camerad/cameras/camera_common.cc

```diff
@@ -350,21 +347,27 @@ static void driver_cam_auto_exposure(CameraState *c, SubMaster &sm) {
   static const bool is_rhd = Params().getBool("IsRHD");
   struct ExpRect {int x1, x2, x_skip, y1, y2, y_skip;};
   const CameraBuf *b = &c->buf;
-#ifndef QCOM2
+
   bool hist_ceil = false, hl_weighted = false;
+  int x_offset = 0, y_offset = 0;
+  int frame_width = b->rgb_width, frame_height = b->rgb_height;
+#ifndef QCOM2
```

Comment:
Still an ifdef here

Reply:
analog_gain is only declared in the header file of qcom2,maybe we can fix this next time